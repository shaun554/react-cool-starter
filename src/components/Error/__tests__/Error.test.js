import React from 'react';
import renderer from 'react-test-renderer';
import { StaticRouter } from 'react-router-dom';

import Error from '../index';

describe('<Error />', () => {
  test('renders', () => {
    const mockData = { message: "Hi I'm error" };
    const tree = renderer
      .create(
        <StaticRouter context={{}}>
          <Error error={mockData} />
        </StaticRouter>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
