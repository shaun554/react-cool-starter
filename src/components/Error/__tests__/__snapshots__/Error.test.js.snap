// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> renders 1`] = `
<div
  className="Error"
>
  <p>
    Oops! 
    Hi I'm error
  </p>
</div>
`;
